src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -106,7 +112,25 @@
     @Argument(doc = "Stop after writing this many sequences.  For testing.")
     public int NUM_SEQUENCES = Integer.MAX_VALUE;
 
+    @Argument(shortName = "AN", doc = "Optional file containing the alternative names for the contigs. "
+            + "First column is the original name, the second column is an alternative name. "
+            + "One contig may have more than one alternative name. " ,
+            optional=true)
+    public File ALT_NAMES = null;
+
     private final MessageDigest md5;
+    
+    /** 
+     * Regular expression defined in the sam spec. Any alternative contig should match this regular expression 
+     * TODO: replace the pattern with a constant : see https://github.com/samtools/htsjdk/pull/956/files
```

Comment:
Is this supposed to be the same as `SAMSequenceRecord.ALTERNATIVE_SEQUENCE_NAME_REGEXP`? It's not the same, it has a `|` character in it.

If it is supposed to be the same, you should make the one in `SamSequenceRecord` public and use it instead of defining another copy here.